var/spack/repos/builtin/packages/amrex/package.py

```diff
@@ -158,7 +161,6 @@ def cmake_args(self):
                                      'linear_solvers'),
             self.define_from_variant('AMReX_AMRDATA', 'amrdata'),
             self.define_from_variant('AMReX_PARTICLES', 'particles'),
-            self.define_from_variant('AMReX_SUNDIALS', 'sundials'),
```

Comment:
This doesn't seem to be conditional on versions. If that's correct we should either completely remove the variant or have something like:
```python
if self.spec.satisfies('@:20.11'):
     args.append(self.define_from_variant('AMReX_SUNDIALS', 'sundials'))
```

Reply:
There are two versions of `cmake_args` depending on the amrex version to build. `AMReX_SUNDIALS` was removed from the version of `cmake-args` used for >= 20.11.

Reply:
Thanks, it wasn't evident from Github's diff but obviously you're correct.